### ISSUE TITLE
Create mock API to return geographies data

### ISSUE THREAD

**Issue by kdutia** [climatepolicyradar/cpr-issues#70]
For prototype, we'll serve data about geographies in an API endpoint which is served by static data within the Python code.

**Request template**
``` 

```

**Comment by kdutia** [comment]
@eurolife @chrisaballard how does this look?

**Comment by eurolife** [comment]
@kdutia Would I be able to do a request to just /geographies to get them all?

**Comment by kdutia** [comment]
@eurolife yes, for sure. Would you still want the filtering option? And is the response format ok?

**Comment by eurolife** [comment]
@kdutia Yes definitely the filtering option, and the format is good, I can work with that.

**Comment by kdutia** [comment]
@eurolife update the spec. Thanks!
